dask/array/creation.py

```diff
@@ -1017,7 +1024,14 @@ def pad_reuse(array, pad_width, mode, *args):
     return result
 
 
-def pad_stats(array, pad_width, mode, *args):
+def _round_if_needed(arr, dtype):
+    if np.issubdtype(dtype, np.integer):
+        rint(arr, out=arr)
```

Comment:
Slight preference to avoid using `out=arr` here. It doesn't make things more efficient, and I find it harder to read (I have to know that `rint` is a ufunc, and that the `out` parameter means I'm mutating `arr`).

Reply:
I was wondering whether to use the out=arr functionality. Personally I don't know what the performance gain is to doing it this way. I'll just use the `out=rint(arr)` instead.